tests/rt_cascading.erl

```diff
@@ -299,7 +321,7 @@ circle_test_() ->
             ?assertEqual(undefined, proplists:get_value(expect_seq, SinkData))
```

Comment:
We should probably use `rt:wait_until` here instead of blindly hoping that this condition exists after 1 second.